src/System/Taffybar.hs

```diff
@@ -175,12 +182,15 @@ data TaffybarConfig =
                  , endWidgets :: [IO Widget] -- ^ Widgets that are packed from right-to-left in the bar
                  }
 
+type TaffybarConfigEQ = (TaffybarConfig, StableName TaffybarConfig)
```

Comment:
I assume you are using the `StableName` for equality tests later.  Is it possible to just do comparisons on `(screenNumber, monitorNumber)` instead?  Or is that not unique enough?  I can imagine someone might want to put two bars on one screen, but is that even possible right now?  Would mixing in the `barPosition` field be enough to get a unique key without requiring stable names?